geo_extensions/transformations/general.py

```diff
@@ -22,3 +24,31 @@ def drop_z_coordinate(polygon: Polygon) -> TransformationResult:
             for interior in polygon.interiors
         ],
     )
+
+
+def round_points(ndigits: SupportsIndex) -> Transformation:
+    """Create a transformation that rounds polygon points to a given number of
+    digits.
+
+    :returns: a callable transformation using the passed parameters
+    """
+
+    def round_points(polygon: Polygon) -> TransformationResult:
```

Comment:
Is redefining the outside's "round_points" okay here? I normally have the internal as `_round_points` or something to keep them separate